app/controllers/api/v2/infrastructures_controller.rb

```diff
@@ -149,10 +149,11 @@ def profile_by_cluster_name
 
   def profile_by_app_group_name
     @app_group = AppGroup.find_by(
-      name: params[:app_group_name]
+      name: params[:app_group_name], 
+      status: "ACTIVE"
     )
 
-    if @app_group.blank? || !@app_group.ACTIVE?
+    if @app_group.blank? || @app_group.status != "ACTIVE"
```

Comment:
we don't need the `@app_group.status != "ACTIVE"` since it already filter on find_by process.

Reply:
ack,